src/wp-includes/formatting.php

```diff
@@ -5847,36 +5847,32 @@ function wp_spaces_regexp() {
 }
 
 /**
- * Prints the important emoji-related styles.
+ * Enqueue the important emoji-related styles.
  *
- * @since 4.2.0
+ * @since 6.4.0
  */
-function print_emoji_styles() {
-	static $printed = false;
-
-	if ( $printed ) {
-		return;
+function enqueue_emoji_styles() {
+	$action = is_admin() ? 'admin_print_styles' : 'wp_print_styles';
+	if ( has_action( $action, 'print_emoji_styles' ) ) {
+		remove_action( $action, 'print_emoji_styles' );
+		$emoji_styles = '
```

Comment:
This can take advantage of language injection in PhpStorm:

```suggestion
		$emoji_styles = /* language=CSS */ '
```

![image](https://github.com/WordPress/wordpress-develop/assets/134745/097113b3-401d-401d-8fb9-9b61c9209ee0)